src/app/public/modules/split-view/split-view-drawer.component.ts

```diff
@@ -0,0 +1,207 @@
+import {
+  AfterViewInit,
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  ElementRef,
+  HostListener,
+  Input,
+  OnDestroy,
+  OnInit
+} from '@angular/core';
+
+import {
+  SkyCoreAdapterService,
+  SkyMediaBreakpoints,
+  SkyMediaQueryService
+} from '@skyux/core';
+
+import {
+  Observable,
+  Subject
+} from 'rxjs';
+
+import {
+  SkySplitViewMediaQueryService
+} from './split-view-media-query.service';
+
+import {
+  SkySplitViewService
+} from './split-view.service';
+
+let nextId = 0;
+
+@Component({
+  selector: 'sky-split-view-drawer',
+  templateUrl: 'split-view-drawer.component.html',
+  styleUrls: ['split-view-drawer.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  providers: [
+    SkySplitViewMediaQueryService,
+    { provide: SkyMediaQueryService, useExisting: SkySplitViewMediaQueryService }
+  ]
+})
+export class SkySplitViewDrawerComponent implements AfterViewInit, OnInit, OnDestroy {
+
+  @Input()
```

Comment:
We need to double check ordering here based on our style guide.

Reply:
Done.

Reply:
Getters should go before normal public variables.